mbf_costmap_nav/include/mbf_costmap_nav/free_pose_search.h

```diff
@@ -0,0 +1,199 @@
+/*********************************************************************
+ *
+ * Software License Agreement (BSD License)
+ *
+ *  Copyright (c) 2023, Willow Garage, Inc.
```

Comment:
F: not sure what the copyright should be, but for sure not Willow Garage!!!
@spuetz , @RainerKuemmerle , what to copyright should set here?
Rapyuta Robotics Co., Ltd.?
Or @renan028's name?

Reply:
fixed following the template of others